Celeste.Mod.mm/MonoModRules.cs

```diff
@@ -2408,5 +2411,29 @@ bgt.un.s	ldloca.s
                 PostProcessType(modder, nested);
         }
 
+        public static void PatchEntityListUpdate(ILContext context, CustomAttribute attrib) {
+            TypeDefinition Entity = MonoModRule.Modder.FindType("Monocle.Entity").Resolve();
+            MethodDefinition entity_UpdatePreceder = Entity.FindMethod("UpdatePreceder");
+            MethodDefinition entity_UpdateFinalizer = Entity.FindMethod("UpdateFinalizer");
+
+            ILCursor cursor = new ILCursor(context);
+            ILLabel activeBranchLabel = cursor.DefineLabel();
+            ILLabel loopBranchLabel = null;
+            if (!cursor.TryGotoNext(MoveType.Before, instr=>instr.MatchBr(out loopBranchLabel))) {
+                throw new Exception("Label not found. This is a temporary error that shouldn't be shown to you unless you know what you're doing.");
+            }
```

Comment:
Why not use a simple `GotoNext` which is already throwing? Also, please add spaces around the `=>`

Reply:
ah I see I forgot to remove the "temporary error", great job Viv